examples/gzip/Dockerfile-service

```diff
@@ -0,0 +1,9 @@
+FROM debian:bullseye
```

Comment:
if we use debian - and im all in favour of that - we should use stable - ie `buster` - also rather use `buster-minimal`

Reply:
sorry - that should be `buster-slim`

Reply:
OK.